paradise_papers_search/requirements/dev.txt

```diff
@@ -0,0 +1 @@
+-r base.txt
```

Comment:
Add this file in case we need a debug dependency locally. What you think? Should I delete this?  🤔

Reply:
Leave it, not gonna hurt anyone to have it there